src/rust-crypto/index.ts

```diff
@@ -197,6 +201,14 @@ async function initOlmMachine(
                 // It will be retried by the sdk.
```

Comment:
No, the sdk will retry later (the KeyQuery request will be in outgoing request). It's an optimisation to avoid having things untrusted for a split moment (shields flickering). So it's ok if it doesn't work.

It's a bit more annoying for migrating local trust, as if it fails we wont migrate local trust. But worst case you'll have to verify manually after? and there is retry now for requests.

Reply:
Ok so I changed the code to retry until success.
I created another issue for the `requestKeyBackupVersion` call similar problem https://github.com/element-hq/element-web/issues/27196